docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -86,6 +86,37 @@ The `-e` option is to ensure the official Azure CLI package is installed. This c
 
 You can now run the Azure CLI with the `az` command from either Windows Command Prompt or PowerShell.
 
+## Enable Tab Completion on PowerShell
+
+When typing in a command followed by a parameter, pressing the tab key will trigger the completion. This saves you time and effort in remembering specific arguments or looking them up in the reference documentation. This capability applies to commands and parameters and also extends to values such as account name and subscription name.
+
+To enable tab completion for the Azure CLI on PowerShell, you need to have version 2.49 or higher installed.
+
+To add tab completion, create or edit the profile stored in the variable `$PROFILE`. The simplest way is to run `notepad $PROFILE` in PowerShell. For more information, see [How to create your profile](/powershell/module/microsoft.powershell.core/about/about_profiles#how-to-create-a-profile) and [Profiles and execution policy](/powershell/module/microsoft.powershell.core/about/about_profiles#profiles-and-execution-policy).
+
+Add the following code to your profile:
```

Comment:
```suggestion
[!Note]
Tab completion in PowerShell requires Azure CLI 2.49 or higher.

When tab completion is enabled, you can press <kbd>Tab</kbd> key to trigger it. Tab completion applies to commands and arguments and also extends to values such as resource group name. This saves your time and effort in remembering them or looking them up in the reference documentation. 

To enable tab completion in PowerShell, create or edit the profile file denoted by the variable `$PROFILE`. The simplest way is to run `notepad $PROFILE` in PowerShell. For more information, see [How to create your profile](/powershell/module/microsoft.powershell.core/about/about_profiles#how-to-create-a-profile) and [Profiles and execution policy](/powershell/module/microsoft.powershell.core/about/about_profiles#profiles-and-execution-policy).

Then add the following code to your profile:
```

Reply:
I haven't looked into whether account name or subscription name can be auto-completed. They are also not used very frequently. Resource group name is a more general one.